rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -30,6 +30,18 @@ exception AdminException {
   2: required AdminErrorCode errorCode,
 }
 
+struct AddDBRequest {
+  # the db to add. If it is not a master, you must provide upstream info
+  1: required string db_name,
+  2: optional string db_role = "SLAVE",
```

Comment:
Sure, lets make it so that we launch as slave, since the user can use admin handler to change the role manually